clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -312,7 +313,9 @@
             EligibleLeadersNotAvailableException::new),
     ELECTION_NOT_NEEDED(84, "Leader election not needed for topic partition", ElectionNotNeededException::new),
     NO_REASSIGNMENT_IN_PROGRESS(85, "No partition reassignment is in progress.",
-            NoReassignmentInProgressException::new);
+            NoReassignmentInProgressException::new),
+    GROUP_SUBSCRIBED_TO_TOPIC(86, "The consumer group is actively subscribed to the topic",
```

Comment:
Might be worth adding a little more detail about the context for this error.

Reply:
Ack. I will make it better.